clipmenu

```diff
@@ -33,27 +48,30 @@ fi
 
 # Blacklist of non-dmenu launchers
 launcher_args=(-l "${CM_HISTLENGTH}")
-if [[ "$CM_LAUNCHER" == fzf ]]; then
-    launcher_args=()
-fi
 
-# rofi supports dmenu-like arguments through the -dmenu flag
-[[ "$CM_LAUNCHER" == rofi ]] && set -- -dmenu "$@"
+if (( CM_INPUT_CLIP )); then
+    chosen_line="$( cat )"
```

Comment:
Just `read` -- it can't be more than one line anyway, so no need to fork `cat`. :-)